io.github.wdaniau.fstl.yml

```diff
@@ -0,0 +1,25 @@
+id: io.github.wdaniau.fstl
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+command: fstl-e
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=home
+  - --filesystem=xdg-run/gvfs # make GnomeVFS accessible
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --device=all
```

Comment:
why do you need `device=all`?

Reply:
you're right it's probably not necessary, I'll supress it.